.github/workflows/ci_support_sphere_flutter.yml

```diff
@@ -0,0 +1,32 @@
+name: Frontend CI
+
+on:
```

Comment:
Could you make this workflow to run only for changes within `src/support_sphere`

Reply:
Also, probably should run on push to main as well

Reply:
Updated